CanLII.js

```diff
@@ -76,7 +71,7 @@ function scrape(doc, url) {
 		newItem.firstPage = reporterDetails[3];
 	}
 	
-	newItem.court = text('#breadcrumbs span', 2);
+	newItem.court = text('#breadcrumbs span.nowrap', 2);
```

Comment:
Probably safer to go with `#breadcrumbs span[itemprop="name"]` here — or even `#breadcrumbs *[itemprop="name"]` — rather than something presentational.